src/lib.rs

```diff
@@ -1,4 +1,7 @@
-//! A basic ZipReader/Writer crate
+//! An ergonomic API for reading and writing ZIP files.
+//!
+//! The current implementation is based on [PKWARE's APPNOTE.TXT v6.3.9](https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT)
+// TODO: Decide on the crate's bias: Do we prioritise permissiveness/correctness/speed/ergonomics?
```

Comment:
Do we need to include this TODO in the code? This seems like it would be better as an issue

Reply:
It'd be good to create an issue, yea. I'd like to keep the TODO though, since it's relevant to expanding the crate description, and would be useful to someone looking at that section of the code.

We could add a reference to the GH issue on this line?